### DISCUSSION TITLE
Problem with ZIP of latest release 2.8.6.2

### DISCUSSION THREAD

**Discussion by michaelpiche** [LibreBooking/librebooking#433] [General]
I'm unable to unzip the 2.8.6.2 (GA4 Fix) release ( file librebooking-2.8.6.2-ga4-fix.zip on this page https://github.com/LibreBooking/app/releases/tag/2.8.6.2  ) . I got an error that the file is not valid.
Also, the file size doesn't seem to be ok. Last release is 76.9 MB vs 7.43 MB for 2.8.6.1
Thanks !

**Comment by JohnVillalovos** [comment]
It worked for me.

```
$ wget https://github.com/LibreBooking/app/releases/download/2.8.6.2/librebooking-2.8.6.2-GA4-Fix.zip
<snip>
$ ls -l librebooking-2.8.6.2-GA4-Fix.zip
-rw-r--r--. 1 jlvillal jlvillal 80,648,817 Aug 19 04:30 librebooking-2.8.6.2-GA4-Fix.zip
$ sha256sum librebooking-2.8.6.2-GA4-Fix.zip
30beba7657eeb37e0d3c4a2b410e8ffa339431e67a49bd8d8b3339d825210b18  librebooking-2.8.6.2-GA4-Fix.zip
$ unzip -t librebooking-2.8.6.2-GA4-Fix.zip
<snip>
No errors detected in compressed data of librebooking-2.8.6.2-GA4-Fix.zip.
```

**Comment by michaelpiche** [comment]
I've finally been able to unzip the file with 7-zip. Windows built-in zip tool is not able to unzip this file. It doesn't work neither on my colleague computer.
